docker/prometheus/start.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+# Note: The Stackdriver cart will write to the arbitrary us-central1-f
+
+if [ ! -z "$PROMETHEUS_CONF" ]
+then
+    echo "${PROMETHEUS_CONF}" | base64 -d > /prometheus/prometheus.yml
+fi
+
+/prometheus/prometheus \
+    --config.file=/prometheus/prometheus.yml \
+    --storage.tsdb.path=/prometheus \
+    --web.console.libraries=/prometheus/console_libraries \
+    --web.console.templates=/prometheus/consoles \
+    --web.listen-address=127.0.0.1:9090 &
+
+/go/bin/stackdriver-prometheus-sidecar \
+    --stackdriver.project-id=$GOOGLE_CLOUD_PROJECT \
```

Comment:
Where does this var come from?  Should we check that this exists first?

Reply:
In GCP this comes through the startup procedure of the container os by default. Or you set it if you want to log to another gcp project.
If you run this local (eg lab) you need to set it when running the docker container. Let me add that to the readme.